sdk/monitor/Azure.Monitor.Query/tests/MonitorQueryTestEnvironment.cs

```diff
@@ -25,5 +28,71 @@ public class MonitorQueryTestEnvironment : TestEnvironment
         public string ConnectionString => GetRecordedOptionalVariable("CONNECTION_STRING");
         public string StorageAccountId => GetRecordedOptionalVariable("STORAGE_ID");
         public string StorageAccountConnectionString => GetRecordedOptionalVariable("STORAGE_CONNECTION_STRING");
+        public string MetricsLocation => GetRecordedVariable("METRICS_LOCATION");
+        public string GetLogsAudience()
+        {
+            Uri authorityHost = new(AuthorityHostUrl);
+
+            if (authorityHost == AzureAuthorityHosts.AzurePublicCloud)
+            {
+                return LogsQueryAudience.AzurePublicCloud.ToString();
+            }
+
+            else if (authorityHost == AzureAuthorityHosts.AzureChina)
+            {
+                return LogsQueryAudience.AzureChina.ToString();
+            }
+
+            else if (authorityHost == AzureAuthorityHosts.AzureGovernment)
+            {
+                return LogsQueryAudience.AzureGovernment.ToString();
+            }
+
+            throw new NotSupportedException($"Cloud for authority host {authorityHost} is not supported.");
+        }
+
+        public string GetMetricsAudience()
+        {
+            Uri authorityHost = new(AuthorityHostUrl);
+
+            if (authorityHost == AzureAuthorityHosts.AzurePublicCloud)
+            {
+                return MetricsQueryAudience.AzurePublicCloud.ToString();
+            }
+
+            else if (authorityHost == AzureAuthorityHosts.AzureChina)
+            {
+                return MetricsQueryAudience.AzureChina.ToString();
+            }
+
+            else if (authorityHost == AzureAuthorityHosts.AzureGovernment)
+            {
+                return MetricsQueryAudience.AzureGovernment.ToString();
+            }
+
+            throw new NotSupportedException($"Cloud for authority host {authorityHost} is not supported.");
+        }
+
+        public string GetIngestionMetricsEndpoint()
```

Comment:
I'm confused by this method name. Should it read GetMetrics**Query**Endpoint instead?

Reply:
this will be removed